liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -1342,7 +1342,11 @@ private void doDevMode() throws MojoExecutionException {
         	
         	// In a multi-module build, dev mode will only be run on one project (the farthest downstream) and compile will
         	// be run on any relative upstream projects. If this current project in the Maven Reactor is not one of those projects, skip it.  
-        	List<MavenProject> relevantProjects = getRelevantMultiModuleProjects(graph);
+        	boolean skipJars = true;
+        	if("spring-boot-project".equals(getDeployPackages())) {
```

Comment:
At this point, is this just an error, independently of whether we think we should skip?    Or are we trying to make this correct imagining a future effort to add support for Spring Boot in dev mode?